edk2toolext/environment/version_aggregator.py

```diff
@@ -15,6 +15,9 @@
 
 
 class version_aggregator(object):
+
+    LOCAL_VERSION = "LOCAL"
```

Comment:
i don't think this belongs here.  LOCAL is a feature of external_dependency and not version aggregator.

Reply:
Agreed. Perhaps instead of Version == LOCAL, it could be Type == LOCAL? (Or Source?)